stac_fastapi/elasticsearch/tests/resources/test_item.py

```diff
@@ -143,6 +143,8 @@ def test_update_item_missing_collection(app_client, load_test_data):
 def test_update_item_geometry(app_client, load_test_data):
     test_item = load_test_data("test_item.json")
 
+    test_item["id"] = "update_test_item_1"
```

Comment:
again, another stop-gap. The index should probably be cleared before each run, and each item should be given a random id.